module/data/activity/base-activity.mjs

```diff
@@ -441,6 +458,26 @@ export default class BaseActivityData extends foundry.abstract.DataModel {
   /*  Helpers                                     */
   /* -------------------------------------------- */
 
+  /**
+   * Add an `canOverride` property to the provided object and, if `override` is `true`, replace the data on the
```

Comment:
I'm becoming a little confused now, but I think this should be `false`, based on my understanding and the code below. I think the term 'override' makes sense from the UX perspective, because we want to essentially 'activate' a given block of data for an Activity. But from an implementation perspective, what's actually happening is that every Activity is, by default, *inheriting* the Item's properties, and we use the `override` flag to turn off that inheritance.

```suggestion
   * Add an `canOverride` property to the provided object and, if `override` is `false`, replace the data on the
```